src/olympia/abuse/actions.py

```diff
@@ -792,6 +810,18 @@ def process_action(self, release_hold=False):
 class ContentActionOverrideApprove(ContentActionTargetAppealApprove):
     description = 'Reported content is within policy, after override'
 
+    @property
+    def previous_decisions(self):
+        """Queryset with previous decisions made that this action would revert."""
+        # We want a queryset to be able to filter later, so create one from the
+        # overriden instance, or just return an empty one.
+        qs = self.decision.__class__.objects.all()
+        return (
+            qs.filter(pk=self.decision.override_of.pk)
+            if self.decision.override_of
+            else qs.none()
+        )
```

Comment:
Does this work?
```suggestion
        return self.decision.__class__.objects.filter(pk=self.decision.override_of_id)    
```

Reply:
It does - 922a8549881a155a95f0808d40a8ca54a8b93e01 (maybe slightly less efficient in the case there is no previous decision - the query will be sent to MySQL which will determine it's an impossible query instead of avoiding the call to MySQL altogether, but it's simpler)